packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -739,11 +747,13 @@ export function getPathOverrideForPackageManager({
 export function getPathForPackageManager({
   cliType,
   lockfileVersion,
+  corepackEnabled,
```

Comment:
the `getPathForPackageManager` function was left in to avoid a breaking change for a publicly exported function, why are we changing the call signature?

Reply:
Yeah, I shouldn't mess with this. Will revert.

Reply:
If we can we should eventually mark `getPathForPackageManager` as deprecated.

Reply:
I marked it as deprecated: https://github.com/vercel/vercel/pull/11596/commits/0ef8ebf65c58f6f98bff8fbc80f7094cd4105681

Reply:
ended up making a backwards-compatible change to the signature in case third party builders are using it: getPathForPackageManager